views/containers/GameLoader.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Game from '../components/Game';
+import LoadingScreen from '../components/LoadingScreen';
+import * as BrowserActions from '../actions/Browser';
+
+class GameLoaderComponent extends React.Component {
+  constructor() {
+    super();
+    this.didLoad = false;
```

Comment:
Either way, when moved the API call to an action creator, this property will live in the store but the principle is the same.